### ISSUE TITLE
CIVIC track errors out with "could not determine adapter type from adapter config snapshot"

### ISSUE THREAD

**Issue by rbuels** [cmdcolin/jbrowse-plugin-civic-api#2]
https://jbrowse.org/code/jb2/v1.3.3/?config=test_data%2Fconfig_demo.json&session=share-TgUe1FpucO&password=f3Iaq

backtrace:

```
rpc.worker.ts:90 rpc-error 4 CoreGetRefNames Error: could not determine adapter type from adapter config snapshot
    at dataAdapterCache.ts:40
    at l (runtime.js:63)
    at Generator._invoke (runtime.js:293)
    at Generator.next (runtime.js:118)
    at r (asyncToGenerator.js:3)
    at u (asyncToGenerator.js:25)
    at asyncToGenerator.js:32
    at new Promise (<anonymous>)
    at asyncToGenerator.js:21
    at Y (dataAdapterCache.ts:30)
(anonymous) @ rpc.worker.ts:90
Promise.catch (async)
(anonymous) @ rpc.worker.ts:72
Promise.then (async)
p.handler @ web-rpc.js:321
```

**Comment by rbuels** [comment]
I looked at the 'CIVIC Cancer variants hg19' track and it seemed to have a BamAdapter adapter type, which seems incorrect

**Comment by cmdcolin** [comment]
This has been part of a larger thing related to  https://github.com/GMOD/jbrowse-components/pull/1967

It can probably be fixed but an adapter can be omitted completely from a snapshot if it just uses only defaults from config which this one does causing errors

**Comment by cmdcolin** [comment]
fixed in 1.0.5 by using a non-default config slot
